jackrabbit-spi2dav/src/main/java/org/apache/jackrabbit/spi2davex/Spi2davexRepositoryServiceFactory.java

```diff
@@ -40,6 +40,12 @@
      */
     public static final String PARAM_REPOSITORY_URI = "org.apache.jackrabbit.spi2davex.uri";
 
+    /**
+     * Optional configuration parameter to allow access to servers with invalid ssl certs.
```

Comment:
You should clarify what invalid really means (expired, wrong host, self-signed, untrusted CA)